[datadog_sensitive_data_scanner_group] Handle sampling rate in scanning group (#3076)
DataDog/terraform-provider-datadog: docs/resources/sensitive_data_scanner_group.md

### Optional

- `description` (String) Description of the Datadog scanning group.

> - `samplings` (Block List, Max: 4) List of sampling configurations per product type for the scanning group. (see [below for nested schema](#nestedblock--samplings))

Reviewer: ```suggestion
- `samplings` (Block List, Max: 4) List of sampling configurations per product type for the scanning group. (See [below for nested schema](#nestedblock--samplings).)
```

Author: This part of the documentation is auto-generated so I do not believe I can modify this

- `query` (String) Query to filter the events.

<a id="nestedblock--samplings"></a>
### Nested Schema for `samplings`

Required:

- `product` (String) Product that the sampling rate applies to. Valid values are `logs`, `rum`, `events`, `apm`.

> - `rate` (Number) Rate at which data in product type will be scanned, as a percentage.

Reviewer: ```suggestion
- `rate` (Number) Percentage rate at which data for the product type is scanned.
```
